examples/todo-jwt/src/index.ts

```diff
@@ -13,6 +18,8 @@ export async function main(options: ApplicationConfig = {}) {
 
   const url = app.restServer.url;
   console.log(`Server is running at ${url}`);
+  console.log(`Try ${url}/ping`);
```

Comment:
I believe we don't need to add this because the `PingController` does not exist in this example.